common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet\src\Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet.csproj" />
```

Comment:
We do not use `DotNet` in the names of our projects or classes. Please remove this part from the project name.